forgerock-auth/src/test/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticationTests.kt

```diff
@@ -451,5 +452,31 @@ class DeviceBindAuthenticationTests {
         return date.time;
     }
 
+    @Test
+    fun testValidateCustomClaimsForValidClaims() {
+        val testObject = None()
+        assertTrue(testObject.validateCustomClaims(customClaims = mapOf("name" to "demo", "email_verified" to true)))
+    }
+
+    @Test
+    fun testValidateCustomClaimsForInvalidClaims() {
+        val testObject = None()
+
+        assertFalse(testObject.validateCustomClaims(mapOf(JWTClaimNames.SUBJECT to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf(JWTClaimNames.EXPIRATION_TIME to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf(JWTClaimNames.ISSUED_AT to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf(JWTClaimNames.NOT_BEFORE to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf(JWTClaimNames.ISSUER to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf("platform" to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf("challenge" to "demo")))
+        assertFalse(testObject.validateCustomClaims(mapOf("android-version" to "demo")))
```

Comment:
@witrisna if we are removing "platform" and "android-version" , we should remove here too